src/execution/operator/join/physical_range_join.cpp

```diff
@@ -62,7 +62,6 @@ PhysicalRangeJoin::GlobalSortedTable::GlobalSortedTable(ClientContext &context,
                                                         RowLayout &payload_layout, const PhysicalOperator &op_p)
     : op(op_p), global_sort_state(BufferManager::GetBufferManager(context), orders, payload_layout), has_null(0),
       count(0), memory_per_thread(0) {
-	D_ASSERT(orders.size() == 1);
```

Comment:
Does this need to be part of this PR?

Reply:
Original `GlobalSortedTable` of `PhysicalRangeJoin` requires the sort_keys only contain one column. Of course,  `PhysicalRangeJoin` needs to satisfy it. But adding from_left column will break this requirement.

Reply:
Got it.